.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,46 @@
+name: golangci-lint
```

Comment:
This yaml file is taken from [here](https://github.com/golangci/golangci-lint-action).